portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -3,31 +3,87 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Date;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
 
 public class Comment {
     public static List<Comment> COMMENTS = new ArrayList<>();
```

Comment:
reserve UPPER_SNAKE_CASE for "private static final" constants. Might be more appropriate to just use as "comments" here since it is mutable.